lib/tasks/importer.rake

```diff
@@ -9,31 +28,31 @@ namespace :import do
   end
 
   task projects: :environment do
-    LetsFreckle::Project.all.each do |project|
+    freckle_client.get_projects.each do |project|
       p1 = Project.find_or_create_by(id: project.id)
       p1.update_columns(name: project.name)
     end
   end
 
   desc "Import new Freckle entries"
-  task entries: [:users, :projects] do
+  task entries: [:projects, :environment] do
     start_date = Time.now.beginning_of_week.to_date
     end_date = Time.now.end_of_week.to_date
 
     puts "start importing entries from #{start_date} to #{end_date}"
 
-    LetsFreckle::Entry.all(all_pages: true,
-                           from: start_date,
-                           to:  end_date).each do |entry|
-      e1 = Entry.find_or_create_by(id: entry.id)
-      e1.update_columns(
-        description: entry.description,
-        minutes:     entry.minutes,
-        date:        entry.date,
-        user_id:     entry.user_id,
-        project_id:  entry.project_id
-      )
+    result = freckle_client.get_entries(from: start_date, to: end_date)
+    save_entries_for(result)
+
+    if result.try(:last_page)
+      last_page = result.last_page.match(/page=(\d+)/)[1].to_i
+      [2..last_page].each do |page|
+        result = freckle_client.get_entries(from: start_date, to: end_date,
+                                            page: page)
+        save_entries_for(result)
+      end
     end
```

Comment:
@mauro-oto Could you move this code and `#save_entries_for` to a service class? It'll make it easier to test its behavior.